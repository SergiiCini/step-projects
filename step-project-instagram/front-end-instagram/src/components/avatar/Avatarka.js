import React from 'react'
import {Image, Transformation} from 'cloudinary-react';


const Avatarka = ({avatarUrl}) => {

    return (
        <>
            <Image
                key = {avatarUrl}
                cloudName = "dan-insta-step"
                publicId = {avatarUrl}
                width = "80"
                crop = "scale">
            <Transformation height="150" width="150" crop="fill" radius="80" />
            </Image>
        </>
    )
}
export default Avatarka;