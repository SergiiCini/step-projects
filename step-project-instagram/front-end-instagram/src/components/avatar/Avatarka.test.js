
import React from 'react'
import Avatarka from './Avatarka';



describe('Avatarka testing url',()=>{
    let avatarUrl
    beforeEach(()=>{
        avatarUrl ={
            avatarUrl:''
        } 
    })
    test('component with avatar url',()=>{
            const avatarUrl ={
                avatarUrl:'instagram/avatars/user_6_avatar_jbtint'
            } 
            const component = <Avatarka avatarUrl={avatarUrl}/>
            expect(component).toMatchSnapshot()
            expect(avatarUrl).toBeDefined()
    })
})
