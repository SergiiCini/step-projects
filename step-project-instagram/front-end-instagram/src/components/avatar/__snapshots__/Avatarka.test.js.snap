// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatarka testing url component with avatar url 1`] = `
<Avatarka
  avatarUrl={
    Object {
      "avatarUrl": "instagram/avatars/user_6_avatar_jbtint",
    }
  }
/>
`;
